.test-infra/jenkins/job_PostCommit_Python_Chicago_Taxi_Example_Dataflow.groovy

```diff
@@ -19,46 +19,44 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 import CronJobBuilder
+import LoadTestsBuilder
 
-
-// This job runs the Chicago Taxi Example script on Dataflow
-PostcommitJobBuilder.postCommitJob(
-    'beam_PostCommit_Python_Chicago_Taxi_Dataflow',
-    'Run Chicago Taxi on Dataflow',
-    'Google Cloud Dataflow Runner Chicago Taxi Example',
-    this
-) {
-    description('Runs the Chicago Taxi Example on the Dataflow runner.')
+def chicagoTaxiJob = { scope ->
+    scope.description('Runs the Chicago Taxi Example on the Dataflow runner.')
 
     // Set common parameters.
-    commonJobProperties.setTopLevelMainJobProperties(delegate)
+    commonJobProperties.setTopLevelMainJobProperties(scope)
+
+    def pipelineOptions = [
+        num_workers          : 5,
+        autoscaling_algorithm: 'NONE',
+    ]
 
     // Gradle goals for this job.
-    steps {
+    scope.steps {
         gradle {
             rootBuildScriptDir(commonJobProperties.checkoutDir)
+            commonJobProperties.setGradleSwitches(delegate)
             tasks(':sdks:python:test-suites:dataflow:py2:chicagoTaxiExample')
             switches('-PgcsRoot=gs://temp-storage-for-perf-tests/chicago-taxi')
+            switches("-PpipelineOptions=\"${LoadTestsBuilder.parseOptions(pipelineOptions)}\"")
         }
     }
 }
 
+PostcommitJobBuilder.postCommitJob(
+    'beam_PostCommit_Python_Chicago_Taxi_Dataflow',
+    'Run Chicago Taxi on Dataflow',
+    'Google Cloud Dataflow Runner Chicago Taxi Example',
```

Comment:
It's nice to include trigger command in the message (easy to rerun if needed). How about:

Chicago Taxi Example on Dataflow (Run Chicago Taxi on Dataflow)